### PR TITLE
Ontology Modules for Digital Product Passport

### PR DESCRIPTION
**Pull request by RichZele**
This pull request is  ontology modules that are developed for digital product passport.  This is the first step and be will refined and new modules will be added as the development progresses.

### CODE REVIEW

**Comment by RichZele** [comment]
Hi Daniel,
Thank you for your message. The modules are already ready. We are waiting
until the paper is published.
I will commit them soon.

Best regards,
Rahel

**Comment by dgarijo** [comment]
@RichZele any news on the files to be committed?

**Comment by RichZele** [comment]
Hi Daniel,

I am just waiting for the publication of the paper in which we introduced
the dpp ontology (digital product passport).
The paper is submitted to a Q1 journal in Elsevier and it is currently
under review... For confirmation, I put the link here:
https://track.authorhub.elsevier.com/?uuid=42d8ebb4-fd5c-4ddb-857f-94908a1521b2


I hope this is ok.

Kind regards,
Rahel

**Comment by dgarijo** [comment]
ping @RichZele for updates about this

**Comment by davidlehn** [comment]
Ping.  Will close this soon if there's no activity.

**Comment by RichZele** [comment]
Hi, 
I added all the modules to the dpp directory. 
Kindg regards,
Rahel

**Comment by davidlehn** [comment]
This service does not support hosting files.  Please use redirects to your hosting service.  Also missing maintainer information.  Look at how all the other ids are setup with `.htaccess` and usually a `README.md`.  When fixing this, please remove all the content files from git history.

File: dpp/Certification_Module/8_Certification_Module.ttl [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    rdfs:subClassOf dpp:Certification .
+
+dpp:EnvironmentalCertification a owl:Class ;
+    rdfs:label "Environmental Certification"@en ;
+    rdfs:comment "The environmental certifications of a product, e.g., RoHS, REACH (https://www.assent.com/blog/difference-reach-rohs-compliance/)."@en ;
+    rdfs:subClassOf dpp:Certification .
+
+dpp:RegulatoryCompliance a owl:Class ;
+    rdfs:label "Regulatory Compliance"@en ;
+    rdfs:comment "Indicates if a product complies with applicable building codes and regulations."@en ;
```
Comment (TallTed) [inline dpp/Certification_Module/8_Certification_Module.ttl:25]:
```suggestion
    rdfs:comment "Indicates whether a product complies with applicable building codes and regulations."@en ;
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by davidlehn** [comment]
Closing this.  Feel free to open another PR that follows the policy of only adding redirects to resources on other sites.

**Pull request closed without merging**